parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -377,11 +376,11 @@ ensureArtifactStreaming() {
   sudo /opt/acr/tools/overlaybd/config.sh exporterConfig.port 9863
   modprobe target_core_user
   curl -X PUT 'localhost:8578/config?ns=_default&enable_suffix=azurecr.io&stream_format=overlaybd' -O
-  systemctl enable /opt/overlaybd/overlaybd-tcmu.service
-  systemctl enable /opt/overlaybd/snapshotter/overlaybd-snapshotter.service
-  systemctl start overlaybd-tcmu
-  systemctl start overlaybd-snapshotter
-  systemctl start acr-nodemon
+  systemctl link /opt/overlaybd/overlaybd-tcmu.service
+  systemctl link /opt/overlaybd/snapshotter/overlaybd-snapshotter.service
+  systemctlEnableAndStart /opt/overlaybd/overlaybd-tcmu.service 30
```

Comment:
are we sure 30 seconds is enough time to start these units? (only asking since this is top-of-mind right now due to the bootstrap token validation stuff)

Reply:
It's not certain that 30s will be enough always. These helper functions will retry so it _should_  succeed eventually. 
Or I could make it go back to the blocking calls on systemctl start and systemctl enable. Thoughts on either option?

AB E2E should test the systemd status, so it should catch any regular issues.